.github/workflows/integration-test.yml

```diff
@@ -106,5 +106,11 @@ jobs:
     permissions:
       actions: write
     steps:
-      - uses: actions/checkout@v4
-      - uses: gautamkrishnar/keepalive-workflow@v2
+      - name: Re-enable workflow
+        env:
+          GITHUB_TOKEN: ${{ inputs.GITHUB_TOKEN || github.token }}
+        shell: sh
+        run: |
+          workflow="${GITHUB_WORKFLOW_REF%%@*}"
+          workflow="${workflow#${GITHUB_REPOSITORY}/.github/workflows/}"
```

Comment:
I don‘t think I can review this anymore, I stopped using {ba,z}sh too long ago.

That being said, maybe link docs to what these magic incantations mean? :smile: